feat: Act page (GC-12 )

Joaquinb2000 opened the pull request:
<!--- Please provide a general summary of your changes in the title above -->

# Pull Request type

<!-- Please try to limit your pull request to one type; submit multiple pull requests if needed. -->

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

Issue Number: [GC-12](https://inreach.atlassian.net/browse/GC-12?atlOrigin=eyJpIjoiN2MwNmZlNzI4MjhjNDE4MTg1ZTEwNjFkYjk0YTNmYTEiLCJwIjoiaiJ9)

## What is the new behavior?

<!-- Please describe the behavior or changes that are being added by this PR. -->

- Added act page
- Created a Back Home Button component to maintain DRY code in the future
-  Changed previewCard component's 'text' prop to accept either a string or react nodes. Also, previewCard now handles strings by splitting them by the '\n' escape character thus separating each paragraph into a p tag.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

<!-- If this does introduce a breaking change, please describe the impact and migration path for existing applications below. -->

## Other information

<!-- Any other information that is important to this PR, such as screenshots of how the component looks before and after the change. -->


[GC-12]: https://inreach.atlassian.net/browse/GC-12?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

JoeKarow reviewed (changes requested):
Please double check the changes you made to the PreviewCard - the text spacing is off on the carousel
 
<img width="1280" alt="image" src="https://user-images.githubusercontent.com/58997957/218625761-332fb9d0-ac58-4320-a66a-bfca70346065.png">

JoeKarow commented:
#### 📝 Changed routes:


| [**/act**](https://glaad-9ae0s0qxl-weareinreach.vercel.app/act) | [**/**](https://glaad-9ae0s0qxl-weareinreach.vercel.app/) |
|:-:|:-:|
|<a href="https://glaad-9ae0s0qxl-weareinreach.vercel.app/act"><img src="https://images.weserv.nl?url=https://sjc.microlink.io/DxGe4sczv-BYAnLPFsLewyki-eKIH-HA-eZUobKPLO5wPOckLyv3dkP1vUC4M5cwnJMUKSqniXECE-UPm37UAQ.png&w=600" alt="Screenshot of /act" width="300"></a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/v3H6TQk_E_BRDMiaX5bBIG5sakCYobfR9DnDmIsMDnsHVb8aUA9PwEu3kuuylB4sf2z8209I2FGaAErqrXWdSg.png">(view full size)</a> |<a href="https://glaad-9ae0s0qxl-weareinreach.vercel.app/"><img src="https://images.weserv.nl?url=https://sjc.microlink.io/IkUP-waxcYCGOXYME9Za2Ndnd9rxay3AoRTIn2s1XjlsohZ4APvIPYQxxD9cvfHTOxr64FUqlNlGJBcYCDomoQ.png&w=600" alt="Screenshot of /" width="300"></a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/aYyDIzMw1vI4Qkzm8BAMdvVtbyXRndblT5GrXM2ZqxmG5QYfoVfc8xgrXD4aa0aZxO_nJz6ujn3eaHAv8cK-Ew.png">(view full size)</a> |







Commit e225eac0e22ae939982800e94744d9c89a1a260c (https://glaad-9ae0s0qxl-weareinreach.vercel.app).

Joaquinb2000 commented:
> Please double check the changes you made to the PreviewCard - the text spacing is off on the carousel
> 
> <img alt="image" width="1280" src="https://user-images.githubusercontent.com/58997957/218625761-332fb9d0-ac58-4320-a66a-bfca70346065.png">

Sorry, that change was intentional. It was in the MainPage.stories file because I changed the 'storyJoy' key-value pair to substitute all of the whitespaces in the lorem text into newline characters. It happened randomly because it was based on whether a faker-generated number was divisible by two. 

I was testing what would happen if someone tried to leave a bunch of one-word lines because it used to make the cards' width and height go all over the place. I deleted the change in the .stories file because it can cause confusion in the future, sorry again.
